metrics.go

```diff
@@ -567,13 +577,20 @@ func (o *BatchJobMetrics) Merge(other *BatchJobMetrics) {
 		// Use latest timestamp
 		o.CollectedAt = other.CollectedAt
 	}
+	// Job
 	if o.Jobs == nil {
 		o.Jobs = make(map[string]JobMetric, len(other.Jobs))
 	}
-	// Job
 	for k, v := range other.Jobs {
 		o.Jobs[k] = v
 	}
+	// Job Type
```

Comment:
maybe remove the comment or make it more descriptive ?

Reply:
Removed redundant comment